apps/omg_watcher_rpc/lib/web/controllers/block.ex

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Controller.Block do
+  @moduledoc """
+  Operations related to block.
+  """
+
+  use OMG.WatcherRPC.Web, :controller
+
+  alias OMG.WatcherInfo.API.Block, as: InfoApiBlock
+  alias OMG.WatcherRPC.Web.Validator
+
+  @doc """
+  Retrieves a list of most recent blocks
+  """
+  def get_blocks(conn, params) do
+    with {:ok, constraints} <- Validator.BlockConstraints.parse(params) do
+      InfoApiBlock.get_blocks(constraints)
+      |> api_response(conn, :blocks)
```

Comment:
```suggestion
      api_response(InfoApiBlock.get_blocks(constraints), conn, :blocks)
```

Reply:
I changed to 2 pipes instead. More readable.